src/panels/config/lovelace/dashboards/dialog-lovelace-dashboard-detail.ts

```diff
@@ -67,21 +66,25 @@ export class DialogLovelaceDashboardDetail extends LitElement {
         escapeKeyAction
         .heading=${createCloseHeading(
           this.hass,
-          this._params.dashboard
+          this._params.urlPath
             ? this._sidebarTitle ||
-                this.hass!.localize(
+                this.hass.localize(
                   "ui.panel.config.lovelace.dashboards.detail.edit_dashboard"
                 )
-            : this.hass!.localize(
+            : this.hass.localize(
                 "ui.panel.config.lovelace.dashboards.detail.new_dashboard"
               )
         )}
       >
         <div>
           ${this._params.dashboard && !this._params.dashboard.id
-            ? this.hass!.localize(
+            ? this.hass.localize(
                 "ui.panel.config.lovelace.dashboards.cant_edit_yaml"
               )
+            : this._params.urlPath === "lovelace"
```

Comment:
Isn't this a weird way to determine if something is the default? Can't we just attach a new attribute to the object? We are creating it on the fly anyway.

Reply:
Actually, all dashboard configs have a url_path on them. Why do we need one in params ?

Reply:
The default panel doesn't have a dashboard config

Reply:
Aren't you creating a fake one ?

Reply:
I did start that way, but bumped into something, not sure what it was anymore...